frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -267,14 +268,17 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
 
   const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
     return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
+      const nodeLabel = isLabelPresent(node, ocsLabel);
+      if (!nodeLabel) {
+        const patch = [
```

Comment:
Not new, but the `patch` const should be defined in a higher scope since it's a static value.